### DISCUSSION TITLE
Disable client app

### DISCUSSION THREAD

**Discussion by scinos** [vuepress/core#660] [Q&A] [answered]
Is there a way to disable the Client App, and use VuePress only to generate And serve SSR pages?

**Comment by Mister-Hope** [comment, marked as answer]
Currently no, and this is not our project goal.

Also, functions like navbar and sidebar may need javascript enabled and compoent actived to work.

If you don't need navbar and sidebar, just static pages rendered by markdown content, you can follow this idea:

Write a script to handle output. The script should try to handle the output files:

1. remove all the js files
2. use regexp to remove all the script tags in output html files.

After you build a project, run the script and get you want
